core/server/data/schema/schema.js

```diff
@@ -300,5 +300,13 @@ module.exports = {
         created_by: {type: 'string', maxlength: 24, nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    sessions: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        session_id: {type: 'string', maxlength: 32, nullable: false, unique: true},
+        user_id: {type: 'string', maxlength: 24, nullable: false},
+        session_data: {type: 'string', maxlength: 2000, nullable: false},
```

Comment:
One last question to this: how much is stored in this data?
Do you think it might happen that we will store more data in this field in the future?
Just trying to understand and avoid that we have to extend this field in the future :)

Reply:
atm we store something like
```json
{
  "user_id": "123456789012345678901234",
  "ip": "255.255.255.255",
  "origin": "https://something.com",
  "cookie": {
    "maxAge": 10000000,
    "expires": true
  }
}
```
It's possible we will store more, but I wouldn't think _that_ much more

Reply:
If we want to be 100% safe, I can make it the same as other json fields - i'm not sure if that will negatively affect the db performance, which is why i went with lower value :man_shrugging:

Reply:
Okay this has a length of 138, which is not much. I think we are good with 2000 :)

Why is the `user_id` stored in `session_data` and in `session.user_id`? Just curious :)

Reply:
Swedish! :+1: 

So we store the entire json of `req.session` in `session_data` and then use it to repopulate the `req.session` after. we added `user_id` column so that in future we can say get all sessions for a user.

I suppose we could remove the user_id from the session_data and just add it manually - i'll look at doing that :)

Reply:
Okay nice, doesn't matter that much i think, but a tiny detail. Alriightty